src/docs/content/reference/current/core/injection/index.md

```diff
@@ -58,6 +58,11 @@ The building blocks for open model profile injection are:
 7. `rampUsersPerSec(rate1).to(rate2).during(duration).randomized`: Injects users from starting rate to target rate, defined in users per second, during a given duration. Users will be injected at randomized intervals.
 8. `stressPeakUsers(nbUsers).during(duration)`: Injects a given number of users following a smooth approximation of the [heaviside step function](http://en.wikipedia.org/wiki/Heaviside_step_function) stretched to a given duration.
 
+{{< alert tip >}}
+Fractional user injection rates are allowed. A common usecase is if you want to express the injection in terms of requests per second and not users per second.  
```

Comment:
No found of this. It introduces a confusing between users per sec and requests per sec.
This feature is not directly related to rps.